Makefile

```diff
@@ -1,4 +1,5 @@
 -include /opt/build-harness/Makefile.prow
+-include Makefile.prow
```

Comment:
`-include /opt/build-harness/Makefile.prow` twice? and does it work when running locally?

Reply:
Yeah it did work. Though you make a good point that I don't need the first entry.
I'll try removing it and make sure it works